mantle/cmd/ore/ibmcloud/upload.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2021 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ibmcloud
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdUpload = &cobra.Command{
+		Use:   "upload",
+		Short: "Create IBMCloud images",
+		Long: `Upload CoreOS image to IBMCloud cloud object storage S3 bucket.
+
+Supported source format is RAW.
+`,
+		Example: `  ore ibmcloud upload --region=us-east \
+	  --cloud-object-storage=coreos-dev-image-ibmcloud \
+	  --bucket=coreos-dev-image-ibmcloud-us-east
+	  --file="/home/.../coreos_production_qcow_image.qcow2"`,
```

Comment:
nit: the example can point to `raw` instead of `qcow2`

Reply:
/lgtm

Reply:
so the example cited here is for IBMCloud (x86) for which the default is qcow. the raw image is related to powervs which is a specific case so i didn't put that here.

Reply:
fixed the typo from raw -> qcow

Reply:
when we add in support for building OVAs , i will update the documentation here too.